core/admin_menu/class-btm-admin-page-task-single.php

```diff
@@ -56,9 +56,11 @@ public function on_hook_page_render_task_single(){
 			}
 		}
 		$dao = BTM_Task_Single_View_Dao::get_instance();
-		$data = $dao->get_task( $task_id );
-		$data = $data[0];
-		$task_status = $data->get_status()->get_value();
+		$task = $dao->get_task( $task_id );
+		if( $task === false ){
+			throw new InvalidArgumentException( 'Wrong task id - ' . $task_id );
+        }
```

Comment:
@eriktad fix spacing.

Reply:
I think it is GitHub code formatting because in my local file all ok!...